lib/inquisition/logger.rb

```diff
@@ -0,0 +1,7 @@
+module Inquisition
+  class Logger
+    def self.log(log)
```

Comment:
I think we need something similar to sidekiq logger
https://github.com/mperham/sidekiq/blob/455e9d56f46f0299eaf3b761596207e15f906a39/lib/sidekiq.rb#L197
https://github.com/mperham/sidekiq/blob/455e9d56f46f0299eaf3b761596207e15f906a39/lib/sidekiq/logger.rb
What do you think?